grammar.js

```diff
@@ -47,6 +49,8 @@ module.exports = grammar({
     up: $ =>        seq('Up',        optional($.speed), optional($.integer)),
     pageup: $ =>    seq('PageUp',    optional($.speed), optional($.integer)),
     pagedown: $ =>  seq('PageDown',  optional($.speed), optional($.integer)),
+    wait: $ =>      seq('Wait', optional($.waitOn), optional($.speed), $.regex),
```

Comment:
Should we rename `$.speed` to duration?

For we do, it'll be valid for both Wait and the other commands that use it (e.g. Type).

I think `duration` semantically makes more sense anyway 🤔